xml/systemd.xml

```diff
@@ -1949,82 +1726,61 @@ WantedBy=multi-user.target
+     class="daemon">dbus</systemitem> is neither recommended nor supported.
+   </para>
+   <para>
+    Updating the <systemitem>dbus</systemitem> or <systemitem>dbus</systemitem>-related packages requires a reboot.
+    When in doubt whether a reboot is necessary, run the <command>sudo zypper ps -s</command>.
+    If <literal>dbus</literal> appears among the listed services, you need to reboot the system.
+   </para>
+   <para>
+    Keep in mind that <systemitem>dbus</systemitem> is updated even when automatic updates are configured to skip the packages that require reboot.
    </para>
-
-    <para>
-     Terminating <systemitem class="daemon">dbus</systemitem> or restarting it
-     in the running system is similar to an attempt to terminate or restart pid
-     1. It will break &systemd; client/server communication and make most &systemd;
-     functions unusable.
-    </para>
-
-    <para>
-     Therefore, terminating or restarting
-     <systemitem
-     class="daemon">dbus</systemitem> is neither recommended
-     nor supported.
-    </para>
-
-    <para>
-     Updating the <systemitem>dbus</systemitem> or
-     <systemitem>dbus</systemitem>-related packages requires a reboot. When in
-     doubt whether a reboot is necessary, run the <command>sudo zypper ps
-     -s</command>. If <literal>dbus</literal> appears among the listed
-     services, you need to reboot the system.
-    </para>
-
-    <para>
-     Keep in mind that <systemitem>dbus</systemitem> is updated even when
-     automatic updates are configured to skip the packages that require reboot.
-    </para>
-
   </sect2>
 
-<!-- debug section should be at the end to be more positive -->
+  <!-- debug section should be at the end to be more positive -->
 
   <sect2 xml:id="sec-boot-systemd-basics-services-debugging">
    <title>Debugging services</title>
    <para>
-    By default, &systemd; is not overly verbose. If a service was started
-    successfully, no output will be produced. In case of a failure, a short
-    error message will be displayed. However, <command>systemctl
-    status</command> provides means to debug start-up and operation of a
-    service.
+    By default, &systemd; is not overly verbose.
+    If a service was started successfully, no output will be produced.
+    In case of a failure, a short error message will be displayed.
+    However, <command>systemctl status</command> provides means to debug start-up and operation of a service.
    </para>
    <para>
-    &systemd; comes with its own logging mechanism (<quote>The Journal</quote>)
-    that logs system messages. This allows you to display the service messages
-    together with status messages. The <command>status</command> command works
-    similar to <command>tail</command> and can also display the log messages in
-    different formats, making it a powerful debugging tool.
+    &systemd; comes with its own logging mechanism (<quote>The Journal</quote>) that logs system messages.
```

Comment:
I am not sure that "The Journal" is a helpful name for this. As I understand it, the module is called `systemd-journald`, and it's accessed with `journalctl`.